table/queries/derived_satellite_scans.sql

```diff
@@ -0,0 +1,205 @@
+                      FROM UNNEST(answers) a)
+                      THEN CONCAT("❗️answer:cert_not_for_domain:", AnswersSignature(answers))
+                -- We check AS after cert because we've seen (rare) cases of blockpages hosted on the ISP that also hosts Akamai servers.
+                WHEN (SELECT LOGICAL_OR(answer.matches_control.asn)
+                      FROM UNNEST(answers) answer)
+                      THEN "✅answer:matches_asn"
+                ELSE CONCAT("❗️answer:not_validated:", AnswersSignature(answers))
+            END
+        )
+    END
+);
+
+
+# BASE_DATASET and DERIVED_DATASET are reserved dataset placeholder names
+# which will be replaced when running the query
+
+# Increment the version of this table if you change the table in a backwards-incomatible way.
+
+# Rely on the table name firehook-censoredplanet.derived.merged_reduced_scans_vN
+# if you would like to see a clear breakage when there's a backwards-incompatible change.
+# Old table versions will be deleted.
+CREATE OR REPLACE TABLE `firehook-censoredplanet.DERIVED_DATASET.reduced_satellite_scans_v1`
+PARTITION BY date
+# Column `country_name` is always used for filtering and must come first.
+# `network`, `subnetwork`, and `domain` are useful for filtering and grouping.
+CLUSTER BY country_name, network, subnetwork, domain
+OPTIONS (
+  friendly_name="Reduced Satellite Scans",
+  description="Filtered and pre-aggregated table of Satellite scans to use with the Censored Planed Dashboard"
+)
+AS (
+WITH Grouped AS (
+    SELECT
+        date,
+
+        # As per https://docs.censoredplanet.org/dns.html#id2, some resolvers are named `special` instead of the real hostname.
+        IF(resolver_name="special","special",NET.REG_DOMAIN(resolver_name)) as reg_hostname,
+        resolver_name as hostname,
+        resolver_as_full_name AS network,
+        CONCAT("AS", resolver_asn, IF(resolver_organization IS NOT NULL, CONCAT(" - ", resolver_organization), "")) AS subnetwork,
+        resolver_country AS country_code,
+
+        IF(domain_is_control, "CONTROL", domain) AS domain,
+        IFNULL(domain_category, "Uncategorized") AS category,
+
+        OutcomeString(domain, received_error, received_rcode, answers) as outcome,
+        
+        COUNT(1) AS count
+    FROM `firehook-censoredplanet.BASE_DATASET.satellite_scan`
+    # Filter on controls_failed to potentially reduce the number of output rows (less dimensions to group by).
+    WHERE domain_controls_failed = FALSE
+    GROUP BY date, hostname, country_code, network, subnetwork, outcome, domain, category
+    # Filter it here so that we don't need to load the outcome to apply the report filtering on every filter.
+    HAVING NOT STARTS_WITH(outcome, "setup/")
+)
+SELECT
+    Grouped.* EXCEPT (country_code),
+    IFNULL(country_name, country_code) AS country_name,
+    CASE
+        WHEN STARTS_WITH(outcome, "expected/") THEN 0
```

Comment:
This is where the count is broken. It's not using the emoji

Reply:
sent you https://github.com/censoredplanet/censoredplanet-analysis/pull/169